include/openenclave/host.h

```diff
@@ -317,6 +317,24 @@ void oe_free_key(
     uint8_t* key_info,
     size_t key_info_size);
 
+/**
+ * Start the switchless call manager for the enclave. This should be done
+ * right after enclave creation for any enclave that has switchless ocalls.
+ *
+ * @param enclave The enclave handle.
+ * @param num_host_workers The number of host worker threads that are
+ * dedicated to servicing switchless ocalls.
+ *
+ * @retval OE_OK The switchless manager is successfully started.
+ * @retval OE_UNEXPECTED Something unexpected happened, e.g. the switchless
+ * manager is already started.
+ * @retval OE_INVALID_PARAMETER The enclave or num_host_workers is not valid.
+ * @retval OE_FAILURE The worker threads are not created properly.
+ */
+oe_result_t oe_start_switchless_manager(
```

Comment:
Why is this new API needed?   Where is the new sample code that shows how it is used?  Or is it just used by edger8r generated code?  (I don't see changes to edger8r).   We should not add new APIs that are public and have no sample code.

Reply:
We will use a structure (exposed publicly) `oe_config_context_switchless_t` for the configuration purpose.